test/AssemblySet_basic_test.py

```diff
@@ -3,16 +3,16 @@
 import shutil
 import unittest
 from pprint import pprint
-
 from test import TEST_BASE_DIR
-from test.test_config import get_test_config
```

Comment:
converted the `test_config` file into `conftest`, and renamed `get_test_config` just `test_config`